src/app/common/utils/system-notifications/system-notifications.service.ts

```diff
@@ -11,9 +11,10 @@ export class SystemNotificationsService {
     private readonly snackBar: MatSnackBar,
   ) { }
 
-  public showNotification({ message, dismissBtnText = 'OK' }: IShowNotificationConfig): void {
+  public showNotification({ message, dismissBtnText = 'OK', type="success" }: IShowNotificationConfig): void {
```

Comment:
Also if you will use enum I've mentioned in other comment/file you should use it here instead string.